types/index.d.ts

```diff
@@ -414,3 +426,10 @@ export interface PlaybackController {
   name: string;
   function: RequestHandler;
 }
+
+export class canFulfillIntent {
+  canFulfillIntent: object;
```

Comment:
If I understand correctly, this `canFulfillIntent` sub-object is the Amazon type described [here](
https://developer.amazon.com/docs/custom-skills/understand-name-free-interaction-for-custom-skills.html#response-to-canfulfillintentrequest), yeah? 

It'd be great if we could add that custom interface to alexa.d.ts and reference it here. I think making it clear this is the underlying Amazon response object will help clear up the confusion of why a `canFulfillIntent` object itself has a field called `canFulfillIntent`